python/cugraph/cugraph/structure/property_graph.py

```diff
@@ -497,11 +497,6 @@ def get_vertex_data(self, vertex_ids=None, types=None, columns=None):
         if self.__vertex_prop_dataframe is not None:
             df = self.__vertex_prop_dataframe
             if vertex_ids is not None:
-                if isinstance(vertex_ids, int):
-                    vertex_ids = [vertex_ids]
-                elif not isinstance(vertex_ids,
-                                    (list, slice, self.__series_type)):
-                    vertex_ids = list(vertex_ids)
```

Comment:
I think we should let the  underlying library handle it rather than limiting `location` to these types. 

Also, this fails with GPU objects like `cupy` , `cudf.Series` , for numpy arrays we pay a cost.

Reply:
Fails when fetching a single element.